lib/puppet/provider/cisco_interface_evpn_multisite/cisco.rb

```diff
@@ -0,0 +1,117 @@
+# Dec 2017
```

Comment:
*nit* `December` to make it consistent with the other files.

Reply:
done.